apm-agent-core/src/main/java/co/elastic/apm/agent/impl/ElasticApmTracer.java

```diff
@@ -192,6 +192,10 @@ public void onChange(ConfigurationOption<?> configurationOption, Double oldValue
         assert assertionsEnabled = true;
     }
 
+    public <T> Transaction startRootTransaction(@Nullable ClassLoader initiatingClassLoader) {
```

Comment:
👍 👍 Very useful!! 
Can you do a quick search-and-replace for all text usages of `startTransaction(TraceContext.asRoot(), null` with a call to this one?

Reply:
I think I did that in some other PR, I forgot in which one. Don't want to introduce more merge conflicts than necessary by doing the search and replace again.